install_config/persistent_storage/selector_label_binding.adoc

```diff
@@ -0,0 +1,143 @@
+[[selector-label-volume-binding]]
+= Selector-Label Volume Binding
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+toc::[]
+
+== Overview
+This guide provides the steps necessary to enable binding of *persistentVolumesClaims* (PVC)
+to *persistentVolumes* (PV) via *selector* and *label* attributes.  By implementing
```

Comment:
To match usage in other topics, this:

```
*persistentVolumesClaims* (PVC) to *persistentVolumes* (PV)
```

can just be:

```
persistent volumes claims (PVCs) to persistent volumes (PVs) 
```

and then use PVC and PV (all cap, un-bolded) as needed throughout.